s3/settings.py

```diff
@@ -0,0 +1,29 @@
+"""
+Configuration for the S3 Interface microservice settings.
+"""
+import json
+import logging
+import os
+
+log = logging.getLogger(__name__)
+
+def get_s3_info():
+    vcap_services = os.getenv("VCAP_SERVICES", "")
+    try:
+        key_id = json.loads(vcap_services)["s3"][0]["credentials"]["access_key_id"]
+        secret_key = json.loads(vcap_services)["s3"][0]["credentials"]["secret_access_key"]
+        region = json.loads(vcap_services)["s3"][0]["credentials"]["region"]
+        bucket = json.loads(vcap_services)["s3"][0]["credentials"]["bucket"]
```

Comment:
This has:
 - 4 duplicate calls to json.loads() with the same input
 - 9 duplicate array lookups all looking at the same "credentials" object

This can be cleaned by by simply calling json.loads() once and storing the result in a local variable, then using another local variable to store the "credentials" object like this:
```suggestion
    vcap_services = os.getenv("VCAP_SERVICES", "")
    try:
        vcap_json = json.loads(vcap_services)
        credentials = vcap_json["s3"][0]["credentials"]
    except (json.JSONDecodeError, KeyError) as err:
        # bail if we can't decode the JSON

    key_id = credentials["access_key_id"]
    secret_key = credentials["secret_access_key"]
    region = credentials["region"]
    bucket = credentials["bucket"]
```

Then at this point you should probably ask yourself "Why am I going through the effort of creating 4 local variables and not just returning the entire credentials object?"